rc/filetype/markdown.kak

```diff
@@ -45,12 +45,27 @@ evaluate-commands %sh{
     awk c cabal clojure coffee cpp crystal css cucumber d diff dockerfile elixir erlang fish
     gas go haml haskell html ini java javascript json julia kak kickstart
     latex lisp lua makefile markdown moon objc ocaml perl pug python ragel
-    ruby rust sass scala scss sh swift toml tupfile typescript yaml sql
+    ruby rust sass scala scheme scss sh swift toml ts tupfile typescript yaml sql
   "
   for lang in ${languages}; do
     printf 'add-highlighter shared/markdown/%s region -match-capture ^(\h*)```\h*(%s\\b|\\{[.=]?%s\\})   ^(\h*)``` regions\n' "${lang}" "${lang}" "${lang}"
     printf 'add-highlighter shared/markdown/%s/ default-region fill meta\n' "${lang}"
-    [ "${lang}" = kak ] && ref=kakrc || ref="${lang}"
+    case "${lang}" in
+      clj) ref="clojure" ;;
+      js)  ref="javascript" ;;
+      cr)  ref="crystal" ;;
```

Comment:
I think you forgot to add the new aliases to the list above, so this is dead code.
Actually, it seems ugly and a bit wasteful to add a separate highlighter for each language alias.
How about adding just one highlighter for each supported language:

```kak
for lang in ${languages}; do
  case "${lang}" in
      (clojure) aliases='(clojure|clj)' ;;
      (javascript) aliases='(javascript|js)' ;;
      # ...
      *) aliases=${lang}
  esac
  printf 'add-highlighter shared/markdown/%s region -match-capture ^(\h*)```\h*(%s\\b|\\{[.=]?%s\\})   ^(\h*)``` regions\n' "${lang}" "${aliases}" "${aliases}"
  printf 'add-highlighter shared/markdown/%s/ default-region fill meta\n' "${lang}"
  printf 'add-highlighter shared/markdown/%s/inner region \A```[^\\n]*\K (?=```) ref %s\n' "${lang}" "${lang}"
```

Reply:
Yes, this is a cleaner approach indeed. Will explore and check that everything will work as expected.